script/build_gh_pages.sh

```diff
@@ -22,12 +22,16 @@ $CCSH sonarimport -nc -o codecharta_sonar.cc.json https://sonarcloud.io maibornw
 $CCSH modify --set-root root/maibornwolff-gmbh_codecharta_visualization -o codecharta_sonar_mod.cc.json codecharta_sonar.cc.json
 
 $CCSH merge -o ../visualization/app/codecharta.cc.json codecharta_sonar_mod.cc.json codecharta_git_mod.cc.json -nc
+# Zipped map for pipeline build
+$CCSH merge -o ../visualization/app/codecharta.cc.json codecharta_sonar_mod.cc.json codecharta_git_mod.cc.json
 
 # Map for analysis
-$CCSH sonarimport -nc -o codecharta_sonar_analysis.cc.json https://sonarcloud.io maibornwolff-gmbh_codecharta_analysis 
+$CCSH sonarimport -nc -o codecharta_sonar_analysis.cc.json https://sonarcloud.io maibornwolff-gmbh_codecharta_analysis
 $CCSH modify --set-root root/maibornwolff-gmbh_codecharta_analysis -o codecharta_sonar_mod.cc.json codecharta_sonar_analysis.cc.json
 $CCSH modify --set-root root/analysis -o codecharta_git_mod.cc.json codecharta_git.cc.json
 $CCSH merge -o ../visualization/app/codecharta_analysis.cc.json codecharta_sonar_mod.cc.json codecharta_git_mod.cc.json -nc
+# Zipped map for pipeline build
+$CCSH merge -o ../visualization/app/codecharta_analysis.cc.json codecharta_sonar_mod.cc.json codecharta_git_mod.cc.json
```

Comment:
```suggestion
$CCSH merge -o ../visualization/app/codecharta_analysis.cc.json.gz codecharta_sonar_mod.cc.json codecharta_git_mod.cc.json
```

Reply:
I tried your suggestion and we don't need it. The merge filter puts `.gz` at the end automatically.